books/free-programming-books-langs.md

```diff
@@ -2150,6 +2150,7 @@ That section got so big, we decided to split it into its own file, the [BY SUBJE
 
 * [A Gentle Introduction To Rust](https://stevedonovan.github.io/rust-gentle-intro) - Steve J Donovan
 * [Asynchronous Programming in Rust](https://rust-lang.github.io/async-book)
+* [Easy Rust](https://dhghomon.github.io/easy_rust/) - David McLeod (HTML, PDF)
```

Comment:
```suggestion
* [Easy Rust](https://dhghomon.github.io/easy_rust) - David McLeod (HTML, PDF)
```

Hi, I am a new reviewer so I am not sure if this is correct, could @davorpa approve of this suggestion.

Reply:
> Hi, I am a new reviewer so I am not sure if this is correct, could @davorpa approve of this suggestion.

Not needed since foce a 301 redirect. Lint rule only applies to root urls. Using the url given by browser is always the best option.

![image](https://user-images.githubusercontent.com/3125580/153734354-08f02055-33fd-4f09-86f3-dd772be6a7c7.png)

Reply:
Noted, thanks!